dist/index.d.ts

```diff
@@ -1,14 +1,6 @@
 /// <reference types="react" />
 import React from 'react';
 
-declare type vals = 'top' | 'left' | 'right' | 'bottom';
-interface CoordsTypes {
-    top: vals;
-    left: vals;
-    right: vals;
-    bottom: vals;
-}
-
 declare type integerVals$1 = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10;
```

Comment:
@henryfradley What's the purpose of the `$1` here?

Reply:
strange, I went through and refactored those types to be cleaner and this is gone.